src/FormBuilder/resources/js/form-builder/src/components/onboarding/steps/withText.tsx

```diff
@@ -1,48 +1,70 @@
-                    <Logo/>
+                <div style={{display: 'flex', justifyContent: 'center', margin: '0 auto var(--givewp-spacing-4)'}}>
+                    <Logo />
                 </div>
             )}
-            {!isFirst && !isLast &&(
-                <div style={{display:'flex', backgroundColor: 'var(--givewp-blue-25)', fontSize: '12px',padding:'4px',borderRadius:'2px',justifyContent:'space-between', alignItems: 'center'}}>
+            {!isFirst && !isLast && (
+                <div
+                    style={{
+                        display: 'flex',
+                        backgroundColor: 'var(--givewp-blue-25)',
+                        fontSize: '12px',
+                        padding: '4px',
+                        borderRadius: '2px',
+                        justifyContent: 'space-between',
+                        alignItems: 'center',
+                    }}
+                >
                     <div>{sprintf(__('Step %s of %s', 'give'), stepNumber, stepCount)}</div>
-                    <Button variant='tertiary' className={'js-exit-tour'}>{__('Exit tour', 'give')}</Button>
+                    <Button variant="tertiary" className={'js-exit-tour'}>
+                        {__('Exit tour', 'give')}
+                    </Button>
                 </div>
             )}
-            <h3 style={{
-                fontSize: isFirst || isLast ? '20px' : '16px',
-                margin: 'var(--givewp-spacing-3) 0',
-            }}>{title}</h3>
+            <h3
+                style={{
+                    fontSize: isFirst || isLast ? '20px' : '16px',
+                    margin: 'var(--givewp-spacing-3) 0',
+                }}
+            >
+                {title}
+            </h3>
             <p style={{fontSize: '14px'}}>{description}</p>
         </div>
-    )
-}
+    );
+};
 
 const withText = (steps) => {
-
     return steps.map((step, index) => {
         const stepCountAdjustedForBookends = steps.length - 2;
-        const textContent = <TextContent
-            title={step.title}
-            description={step.text}
-            stepNumber={index}
-            stepCount={stepCountAdjustedForBookends}
-            isFirst={index === 0}
-            isLast={index === steps.length - 1}
-        />;
-        return {...step, ...{
-                text: renderToStaticMarkup(textContent)
-            }}
-    })
-}
+        const textContent = (
+            <TextContent
+                title={step.title}
+                description={step.text}
+                stepNumber={index}
+                stepCount={stepCountAdjustedForBookends}
+                isFirst={index === 0}
+                isLast={index === steps.length - 1}
+            />
+        );
+        const tempContainer = document.createElement('div');
+        render(textContent, tempContainer);
+
+        return {
+            ...step,
+            ...{
+                text: tempContainer.innerHTML,
+            },
```

Comment:
@pauloiankoski Tiny thing. This doesn't need to be a destructured object. It can just be a property after destructuring step:
```suggestion
            text: tempContainer.innerHTML,
```